src/core/tests/preprocess.cpp

```diff
@@ -923,6 +923,25 @@ TEST(pre_post_process, resize_no_model_layout) {
     EXPECT_NO_THROW(p.build());
 }
 
+TEST(pre_post_process, resize_with_bilinear_pillow) {
+    const auto f = create_simple_function(element::f32, Shape{1, 3, 100, 100});
+    auto p = PrePostProcessor(f);
+    p.input().tensor().set_layout("NCHW");
+    p.input().preprocess().resize(ResizeAlgorithm::RESIZE_BILINEAR_PILLOW);
+    EXPECT_NO_THROW(p.build());
+    EXPECT_EQ(f->output().get_partial_shape(), (Shape{1, 3, 100, 100}));
```

Comment:
Can the resized output be expected?
```suggestion
    p.input().preprocess().resize(ResizeAlgorithm::RESIZE_BILINEAR_PILLOW, 200, 300);
    EXPECT_NO_THROW(p.build());
    EXPECT_EQ(f->output().get_partial_shape(), (Shape{1, 3, 200, 300}));
```